index.html

```diff
@@ -57,5 +57,183 @@ <h1 class="self">
+        </div>
+      </div>
+    </div>
+    <div class="card item2">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="card item3">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="card item4">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="card item5">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="card item6">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>    
+
+  </section>
+
+<!--About me Section-->
+
+  <section id="aboutme">
+
+    <div class="aboutwrapper">
+
+      <img src="./image/Mask Group.png" alt="design element" class="top-img">
+      <img src="./image/Group.png" alt="design element" class="bottom-img">
+
+      <div class="aboutbox">
+
+        <div class="about-title">
+          <h2>About me</h2>
+        </div>
+        <div class="descrip">
+          <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
+        </div>
+        <div class="resumebtn">
+          <a href="#">Get my resume</a>
+        </div>
+      </div>
+
+      <div class="skill-card">
+          <div class="icon-box">
+            <div class="icon">
+              <img src="./image/Group (3).png" alt="world icon">
+            </div>
+          </div>
+          <div class="tagbox">
+            <h4>Languages</h4>
```

Comment:
- [ ] It is important that to represent each of the `languages, frameworks and skills` you do this using the semantic `ul & li` tag rather than representing them as `divs & h4`. The div should be the ul tag while each technology should be represented as li tags. :+1: Using semantic tags is good for seo and it assists screen readers and makes your web page more accessible to a wide range of users. :+1: :grin: